kafka/README.md

```diff
@@ -1,220 +1,91 @@
-# CloudEvents Kafka Protocol Binding
+# Kafka Protocol Binding
 
-The impl of Kafka Protocol Biding for CloudEvents.
-
-> See spec [here](https://github.com/cloudevents/spec/blob/master/kafka-protocol-binding.md)
-
-## How to Use
-
-See some examples of how to use with Kafka Consumer and Kafka Producer.
-
-Add the dependency in your project:
+For Maven based projects, use the following to configure the [Kafka Protocol Binding](https://github.com/cloudevents/spec/blob/master/kafka-protocol-binding.md):
 
 ```xml
 <dependency>
     <groupId>io.cloudevents</groupId>
-    <artifactId>cloudevents-kafka</artifactId>
-    <version>1.3.0</version>
+    <artifactId>http-vertx</artifactId>
```

Comment:
uh? Shouldn't it be `cloudevents-kafka`?

Reply:
wtf something went wrong with the rebase